code/modules/mob/living/simple_animal/corpse.dm

```diff
@@ -154,4 +154,18 @@
 	corpsehelmet = /obj/item/clothing/head/bearpelt
 
 /obj/effect/landmark/mobcorpse/russian/ranged
-	corpsehelmet = /obj/item/clothing/head/ushanka
\ No newline at end of file
+	corpsehelmet = /obj/item/clothing/head/ushanka
+
+
+/obj/effect/landmark/mobcorpse/russian/ranged/spetsnaz

```

Comment:
It's for a new Russian spetsnaz mob using pre-existing gear, won't spawn anywhere by default at the moment, might be useful for an away mission though.